code/modules/holiday/holidays.dm

```diff
@@ -366,6 +366,16 @@
 	begin_month = JUNE
 	begin_weekday = SUNDAY
 
+/datum/holiday/moth
+	name = "Moth Week"
+	begin_day = 21
+	begin_month = JULY
+	end_day = 29
```

Comment:
It says during the last full week of July, so these shouldn't be hardcoded days.

Reply:
I have been embarrassed on my own domain (fixing)